Core/App.Web/StartupBase.cs

```diff
@@ -75,13 +82,18 @@ public virtual void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 .UseSwaggerUI(Configuration);
 
             app.UseAuthentication(Configuration);
+            app.UseCorrelationId();
 
             app.UseErrorHandling();
 
+            app.UseAuthentication(Configuration);
```

Comment:
remove this line, it's duplicated with line 84.

Reply:
Sharp!